[2020-09-01T18:06:23.449] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/server/setRedis
request client ip:  ::1
request query:  {}
response time: 1598954783449
err name: Error
err message: node_redis: The SET command contains a invalid argument type.
Only strings, dates and buffers are accepted. Please update your code to use valid argument types.
err stack: Error: node_redis: The SET command contains a invalid argument type.
Only strings, dates and buffers are accepted. Please update your code to use valid argument types.
    at RedisClient.internal_send_command (H:\fancy-gm-cp\node_modules\redis\index.js:834:39)
    at RedisClient.set (H:\fancy-gm-cp\node_modules\redis\lib\commands.js:46:25)
    at H:\fancy-gm-cp\src\config\/redis.js:30:11
    at new Promise (<anonymous>)
    at _callee2$ (H:\fancy-gm-cp\src\config\/redis.js:29:18)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Redis.<anonymous> (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Redis.set (H:\fancy-gm-cp\src\config\redis.js:116:21)
    at _callee2$ (H:\fancy-gm-cp\src\app\service\/server.js:19:15)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-01T19:06:14.922] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/exchange
request client ip:  ::1
request query:  {}
response time: 1598958374921
err name: TypeError
err message: _cdk2.default.findAll is not a function
err stack: TypeError: _cdk2.default.findAll is not a function
    at _callee$ (H:\fancy-gm-cp\src\app\controllers\/cdk.js:11:30)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at findAll (H:\fancy-gm-cp\src\app\controllers\cdk.js:76:25)
    at dispatch (H:\fancy-gm-cp\node_modules\koa-compose\index.js:44:32)
    at next (H:\fancy-gm-cp\node_modules\koa-compose\index.js:45:18)
    at H:\fancy-gm-cp\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\fancy-gm-cp\node_modules\koa-compose\index.js:44:32)
    at H:\fancy-gm-cp\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\fancy-gm-cp\node_modules\koa-router\lib\router.js:351:31)
    at next (H:\fancy-gm-cp\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at _callee2$ (H:\fancy-gm-cp\src\middleware\/index.js:38:11)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at H:\fancy-gm-cp\src\middleware\/index.js:31:2
    at next (H:\fancy-gm-cp\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at _callee$ (H:\fancy-gm-cp\src\middleware\/index.js:29:11)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-01T19:11:43.506] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/exchange?key=9A9NxFxXpx&roleid=121654594564564654&plaform=%E5%AE%89%E5%8D%93&channel=%E4%B9%85%E6%B8%B8
request client ip:  ::1
request query:  {"key":"9A9NxFxXpx","roleid":"121654594564564654","plaform":"安卓","channel":"久游"}
response time: 1598958703505
err name: TypeError
err message: Cannot read property 'type' of undefined
err stack: TypeError: Cannot read property 'type' of undefined
    at CDKService._callee2$ (H:\fancy-gm-cp\src\app\service\/cdk.js:20:9)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-01T19:13:12.525] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/exchange?key=9A9NxFxXpx&roleid=121654594564564654&plaform=%E5%AE%89%E5%8D%93&channel=%E4%B9%85%E6%B8%B8
request client ip:  ::1
request query:  {"key":"9A9NxFxXpx","roleid":"121654594564564654","plaform":"安卓","channel":"久游"}
response time: 1598958792523
err name: ReferenceError
err message: mongodb is not defined
err stack: ReferenceError: mongodb is not defined
    at _callee4$ (H:\fancy-gm-cp\src\app\service\cdk.js:224:17)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at CDKService.<anonymous> (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at CDKService.cdkMutually (H:\fancy-gm-cp\src\app\service\cdk.js:260:29)
    at CDKService._callee2$ (H:\fancy-gm-cp\src\app\service\/cdk.js:28:53)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-01T19:17:18.218] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/exchange?key=fPoM2OR9Gj%20&roleid=1216545945645646541&plaform=%E5%AE%89%E5%8D%93&channel=%E4%B9%85%E6%B8%B8
request client ip:  ::1
request query:  {"key":"fPoM2OR9Gj ","roleid":"1216545945645646541","plaform":"安卓","channel":"久游"}
response time: 1598959038217
err name: TypeError
err message: Cannot read property 'type' of undefined
err stack: TypeError: Cannot read property 'type' of undefined
    at CDKService._callee2$ (H:\fancy-gm-cp\src\app\service\/cdk.js:20:9)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-01T19:52:30.116] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/exchange?key=fPoM2OR9Gj%20&roleid=1216545945645646541&plaform=%E5%AE%89%E5%8D%93&channel=%E4%B9%85%E6%B8%B8
request client ip:  ::1
request query:  {"key":"fPoM2OR9Gj ","roleid":"1216545945645646541","plaform":"安卓","channel":"久游"}
response time: 1598961150115
err name: SequelizeDatabaseError
err message: 对于表"cdk",丢失FROM子句项
err stack: SequelizeDatabaseError: 对于表"cdk",丢失FROM子句项
    at Query.formatError (H:\fancy-gm-cp\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\fancy-gm-cp\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
    at tryCatcher (H:\fancy-gm-cp\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (H:\fancy-gm-cp\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (H:\fancy-gm-cp\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (H:\fancy-gm-cp\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (H:\fancy-gm-cp\node_modules\bluebird\js\release\promise.js:725:18)
    at _drainQueueStep (H:\fancy-gm-cp\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (H:\fancy-gm-cp\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (H:\fancy-gm-cp\node_modules\bluebird\js\release\async.js:102:5)
    at Immediate.Async.drainQueues [as _onImmediate] (H:\fancy-gm-cp\node_modules\bluebird\js\release\async.js:15:14)
    at processImmediate (internal/timers.js:456:21)
*************** error log end ***************

[2020-09-01T19:53:32.189] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/exchange?key=fPoM2OR9Gj%20&roleid=1216545945645646541&plaform=%E5%AE%89%E5%8D%93&channel=%E4%B9%85%E6%B8%B8
request client ip:  ::1
request query:  {"key":"fPoM2OR9Gj ","roleid":"1216545945645646541","plaform":"安卓","channel":"久游"}
response time: 1598961212188
err name: TypeError
err message: Cannot read property 'type' of undefined
err stack: TypeError: Cannot read property 'type' of undefined
    at CDKService._callee2$ (H:\fancy-gm-cp\src\app\service\/cdk.js:24:9)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-01T21:14:28.039] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/cdk/exchange?key=9A9NsBQ6Sr&roleid=111&plaform=%E5%AE%89%E5%8D%93&channel=%E4%B9%85%E6%B8%B8
request client ip:  ::1
request query:  {"key":"9A9NsBQ6Sr","roleid":"111","plaform":"安卓","channel":"久游"}
response time: 1598966068036
err name: CastError
err message: Cast to date failed for value "2020-14-We 05:14:27" at path "receive"
err stack: CastError: Cast to date failed for value "2020-14-We 05:14:27" at path "receive"
    at model.Query.exec (H:\fancy-gm-cp\node_modules\mongoose\lib\query.js:4368:21)
    at model.Query.Query.findOneAndUpdate (H:\fancy-gm-cp\node_modules\mongoose\lib\query.js:3030:8)
    at Function.findOneAndUpdate (H:\fancy-gm-cp\node_modules\mongoose\lib\model.js:2477:13)
    at res (H:\fancy-gm-cp\src\config\/mongo.js:217:14)
    at new Promise (<anonymous>)
    at mongodb._callee11$ (H:\fancy-gm-cp\src\config\/mongo.js:216:19)
    at tryCatch (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (H:\fancy-gm-cp\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\fancy-gm-cp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
*************** error log end ***************

